### PR TITLE
Enable error log for ocs and webdav requests 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9449

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/composables/download/useDownloadFile.ts [hunk 1]
```diff
@@ -29,30 +27,26 @@ export const useDownloadFile = () => {
...
           const signedUrl = await client.signUrl(url)
           triggerDownloadWithFilename(signedUrl, file.name)
           return
         }
       } catch (e) {
         console.error(e)
+        store.dispatch('showErrorMessage', {
+          title: $gettext('Download failed'),
+          desc: $gettext('File could not be located'),
+          error: e
```
Comment (lookacat) [inline packages/web-pkg/src/composables/download/useDownloadFile.ts:47]: shouldn't it be `e.error`? like in the first change?

File: packages/web-app-files/src/helpers/share/link.ts [hunk 2]
```diff
@@ -68,19 +68,26 @@ export const createQuicklink = async (args: CreateQuicklink): Promise<Share> =>
...
-    title: $gettext('The link has been copied to your clipboard.')
-  })
-
-  return link
+    return link
+  } catch (e) {
+    console.error(e)
+    await store.dispatch('showErrorMessage', {
+      title: $gettext('Copy link failed'),
+      error: e
```
Comment (lookacat) [inline packages/web-app-files/src/helpers/share/link.ts:90]: shouldn't it be `e.error`? like in the first change?

Comment (JanAckermann) [reply packages/web-app-files/src/helpers/share/link.ts:90]: what you do you mean with, like in the first change? why should the error object contain an error field?

File: packages/web-pkg/src/composables/download/useDownloadFile.ts (hunk 1 above)
Comment (JanAckermann) [reply packages/web-pkg/src/composables/download/useDownloadFile.ts:47]: what you do you mean with, like in the first change? why should the error object contain an error field?

Comment (lookacat) [reply packages/web-pkg/src/composables/download/useDownloadFile.ts:47]: my bad, its fine

File: packages/web-app-files/src/helpers/share/link.ts (hunk 2 above)
Comment (lookacat) [reply packages/web-app-files/src/helpers/share/link.ts:90]: my bad, its fine

**Review by JammingBen — COMMENTED** [review]
Nice work, code LGTM, the tests I did looked good.

One small nitpick: The success notification has some margin at the bottom, causing it to not be vertically centered anymore.

Pasted Graphic

**Comment by JanAckermann** [comment]
> Nice work, code LGTM, the tests I did looked good.
> 
> One small nitpick: The success notification has some margin at the bottom, causing it to not be vertically centered anymore.
> 
> Pasted Graphic

oh yeah, I need to adjust in the ods

**Review by JammingBen — APPROVED** [review]
👍

**Pull request merged**